public/js/main.js

```diff
@@ -76,7 +76,9 @@ function getTargetFromQuery() {
 
 const connTarget = getTargetFromQuery();
 if (connTarget) {
-  startDebugging(connTarget, actions).then(() => {
+  startDebugging(connTarget, actions).then((tabs) => {
+    actions.newTabs(tabs);
```

Comment:
They never need to be set? The state should be pulled out into whatever toolbox thing we come up with. But this doesn't do any harm.